### PR TITLE
Use HTTPS mirror in `JavaProcessStackTracesMonitorSpec`

### PR DESCRIPTION
**Pull request by lacasseio**
### Context
Fixes https://github.com/gradle/gradle-private/issues/2060

### Gradle Core Team Checklist
- [ ] Verify design and implementation 
- [ ] Verify test coverage and [CI build status](https://builds.gradle.org/project.html?projectId=Gradle_Check&tab=projectOverview&branch_Gradle_Check=lacasseio%2Fuse-to-https-mirror)
- [ ] Verify documentation
- [ ] Recognize contributor in release notes

### CODE REVIEW

**Comment by blindpirate** [comment]
This is unnecessary to be fixed. Take a look at this test, it's just testing extract pid from `ps` logs. It doesn't matter if we're using mirror or not.

File: subprojects/internal-integ-testing/src/test/groovy/org/gradle/integtests/fixtures/timeout/JavaProcessStackTracesMonitorSpec.groovy [hunk 1]
```diff
@@ -35,7 +35,7 @@ class JavaProcessStackTracesMonitorSpec extends Specification {
...
...e.integtest.mirrors.jboss=https://dev12.gradle.org:8081/artifactory/jboss/ -Dorg.gradle.integtest.mirrors.jcenter=https://dev12.gradle.org:8081/artifactory/jcenter -Dorg.gradle.integtest.mirrors.lightbendmaven=https://dev12.gradle.org:8081/artifactory/typesafe-maven-releases -Dorg.gradle.integtest.mirrors.ligthbendivy -Dorg.gradle.integtest.mirrors.mavencentral=https://dev12.gradle.org:8081/artifactory/repo1 -Dorg.gradle.integtest.mirrors.restlet=https://dev12.gradle.org:8081/artifactory/restlet/ -Dorg.gradle.integtest.mirrors.springreleases=https://dev12.gradle.org:8081/artifactory/spring-releases/ -Dorg.gradle.integtest.mirrors.springsnapshots=https://dev12.gradle.org:8081/artifactory/spring-snapshots/ -Dorg.gradle.integtest.multiversion=default -Dorg.gradle.integtest.native.toolChains=default -Dorg.gradle.integtest.versions=latest -Dorg.gradle.native=false -Dorg.gradle.test.maxParallelForks=4 -XX:+HeapDumpOnOutOfMemoryError -Xmx512m -Dfile.encoding=UTF-8 -Duser.country=US -Duser.language=en -Duser.variant -ea -cp /home/tcagent1/.gradle/caches/4.9-20180607113442+0000/workerMain/gradle-worker.jar worker.org.gradle.process.internal.worker.GradleWorkerMain 'Gradle Test Executor 61\'
```
Comment (blindpirate) [inline subprojects/internal-integ-testing/src/test/groovy/org/gradle/integtests/fixtures/timeout/JavaProcessStackTracesMonitorSpec.groovy]: Let's remove the parameter `-Dorg.gradle.integtest.mirrors.gradle`. It seems to be misleading many people. This is just a text block to test extraction of process pid. I copied from `ps aux` output from a build agent.

Comment (lacasseio) [reply subprojects/internal-integ-testing/src/test/groovy/org/gradle/integtests/fixtures/timeout/JavaProcessStackTracesMonitorSpec.groovy]: Great! Thanks a lot @blindpirate for your insight.

**Comment by lacasseio** [comment]
Pre-emptively merging this change, will fix any issue on `release`.

**Pull request merged**